src/main/resources/db/migration/changes/007-create-Instructors.json

```
@@ -0,0 +1,43 @@
+{ "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "007-create-Instructors.json",
```

My bad: i meant this.   I'm going to go ahead and make the change since I goofed on the previous code review:

```suggestion
        "id": "007-create-Instructors",
```